### ISSUE TITLE
Add argument to ignore not yet implemented types

### ISSUE THREAD

**Issue by John0x** [oxidecomputer/progenitor#106]
Hey, I'm getting a bunch of "not yet implemented" errors. (password, timestamp, ipv4...).
Could you maybe add the option to just ignore types that aren't implemented yet?

**Comment by jclulow** [comment]
I think one of the challenges with ignoring unknown input is what should we generate at that point?  Would it be sufficient to fall back on strings, etc.  Can you provide snippets of OpenAPI documents that show what we're failing to interpret?

**Comment by John0x** [comment]
Well, everything is better than just failing completely 😃
Falling back on strings would probably make the most sense tho

Unfortunately, I cannot share much of the document, but here are some snippets:

```json
	"ip": {
		"description": "The IP address of the user at the time the action took place.",
		"example": "127.0.0.1",
		"oneOf": [
			{
				"type": "string",
				"format": "ipv4"
			}
		]
	},
```

```json
	"last_access": {
		"nullable": true,
		"type": "string",
		"format": "timestamp"
	},
```

```json
	"password": {
		"type": "string",
		"description": "Password of the user.",
		"format": "password",
		"example": "password"
	},
```

**Comment by karencfv** [comment]
Heya @John0x! 

I think this issue might be coming from the Typify crate. It looks like support for IP types has just been added -> https://github.com/oxidecomputer/typify/pull/76 but there hasn't been a release yet. Hope this helps!

**Comment by ahl** [comment]
> Well, everything is better than just failing completely 😃

Agreed! As this (and typify) move from prototypes to more robust tools, I've been replacing `todo!`s with reasonable defaults, fallbacks, and explicit errors. It's balanced against other work. Can you share the specific OpenAPI spec that's failing for you?

**Comment by John0x** [comment]
> > Well, everything is better than just failing completely 😃
> 
> Agreed! As this (and typify) move from prototypes to more robust tools, I've been replacing `todo!`s with reasonable defaults, fallbacks, and explicit errors. It's balanced against other work. Can you share the specific OpenAPI spec that's failing for you?

Unfortunately, I cannot share more than the above. But you can probably reproduce it since the api is being generated by https://directus.io/

**Comment by John0x** [comment]
Just tried the most recent master branch and timestamp still breaks the build

**Comment by ahl** [comment]
Ok! Any unrecognized string format should now just be treated as a string. Can you give it another try?

**Comment by John0x** [comment]
I believe it would be better to just print a warning for unsupported stuff. Currently it's failing for:
"Error: servers not presently supported"
"Error: op readItemsAreas: security, unsupported"
